cookiecutter.json

```diff
@@ -35,6 +35,7 @@
   "use_drf": "n",
   "frontend_pipeline": ["None", "Django Compressor", "Gulp", "Webpack"],
   "use_celery": "n",
+  "use_django_rq": "n",
```

Comment:
> Should use_celery and use_django_rq be mutually exclusive, or allow both for advanced setups?

AFAIU, Celery and RQ are solving the same problem, so we should design the questions to avoid having both installed. I would change this question to something like `"background_queue": ["None", "Celery", "Django-RQ"]`. Basically turn 2 Y/N questions into one drop-down question.